Tasks/Trots/Task4/Task4.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python3
+while True:
+    n = (input("Please enter the maximum number of characters per line(must be over 35): ")).strip()
+    if n.isdigit():
+        if int(n) > 35:
+            n = int(n)
+            break
+        else:
+            print("Input Error!!Value must be over 35! Try again!")
+    else:
+        print("Input Error!!Please, enter a numeric value!")
+
+text = ""
+
+with open("text2.txt", 'r', encoding="utf-8") as file:
+    for line in file:
+        text += line

```

Comment:
reading the whole file in memory is not very optimal strategy, since in can be very big